draft-ietf-ohai-ohttp.md

```diff
@@ -132,75 +132,84 @@ them or linked to other requests.
   resource supports.
 
-* The details of an HPKE public key that the oblivious request resource accepts,
-  including an identifier for that key and the HPKE algorithms that are used
-  with that key.
+* The details of an HPKE public key that the Oblivious Gateway Resource
+  accepts, including an identifier for that key and the HPKE algorithms that
+  are used with that key.
 
-* The identity of an oblivious proxy resource that will forward encapsulated
-  requests and responses to a single oblivious request resource. See {{proxy-state}}
-  for more information about the mapping between oblivious proxy and oblivious
-  request resources.
+* The identity of an Oblivious Relay Resource that will accept relay requests
+  carrying an encapsulated request as its content and forward the content in
+  these requests to a single Oblivious Gateway Resource. See {{proxy-state}}
+  for more information about the mapping between Oblivious Relay and oblivious
+  encapsulation resources.
 
 This information allows the client to make a request of an oblivious target
 resource with that resource having only a limited ability to correlate that
 request with the client IP or other requests that the client might make to that
 server.
 
 ~~~ aasvg
-+---------+       +----------+       +----------+    +----------+
-| Client  |       | Proxy    |       | Request  |    | Target   |
-|         |       | Resource |       | Resource |    | Resource |
-+----+----+       +----+-----+       +----+-----+    +----+-----+
-     |                 |                  |               |
-     | Encapsulated    |                  |               |
-     | Request         |                  |               |
-     +---------------->| Encapsulated     |               |
-     |                 | Request          |               |
-     |                 +----------------->| Request       |
-     |                 |                  +-------------->|
-     |                 |                  |               |
-     |                 |                  |      Response |
-     |                 |     Encapsulated |<--------------+
-     |                 |         Response |               |
-     |    Encapsulated |<-----------------+               |
-     |        Response |                  |               |
-     |<----------------+                  |               |
-     |                 |                  |               |
++---------+       +----------+      +----------+      +----------+
+| Client  |       | Relay    |      | Gateway  |      | Target   |
+|         |       | Resource |      | Resource |      | Resource |
++----+----+       +----+-----+      +-------+--+      +----+-----+
+     |                 |                    |              |
+     | Relay           |                    |              |
+     | Request         |                    |              |
+     | [+ Encapsulated |                    |              |
```

Comment:
```suggestion
     | [+ Encrypted    |                    |              |
```

Reply:
I kept Encapsulated here since that's the term we use to describe the Relay Request's content.